kuksa_databroker/databroker/tests/features/current_values.feature

```diff
@@ -35,7 +35,7 @@ Feature: Writing and reading the current value of a VSS Data Entry
         | Vehicle.TraveledDistanceHighRes        | uint64 | 23425462462563924 |
         | Vehicle.CurrentLocation.Longitude      | double |        145.023544 |
         | Vehicle.Speed                          | float  |              45.5 |
-        | Vehicle.IsMoving                       | bool   |              true |
+        | Vehicle.IsMogving                       | bool   |              true |
```

Comment:
Deliberate error to check if it makes build fail, if working I will remove it in a later update